### ISSUE TITLE
pocket query import results in wrong number of favorites

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#7746]
**Describe the bug:**
Importing a pocket query from gc.com results in weird number of favorites stored. Most caches get "0" favs (although they do have favos been given), others do have values, but they differ from the ones on the website.

**To Reproduce:**
- create a pocket query on gc.com, let the results be displayed and sort by favos descending
- import the results into c:geo and sort the list by number of favos descending
- the two lists differ widely
(This was a PQ along a route - not sure if it's limited to this.)

**Expected behavior/state after performing these steps:**
- The two lists should be identical in terms of favos shown.
- Or: If number of favos are not part of the pocket query, default should be 0.

**Version of c:geo used:**
current release 2019.06.06

**Is the problem reproducible:**
Yes

**Screenshots:**
This is a pocket query along a route from the Ruhrgebiet to north of Hamburg. Both screenshots show the top of the list, sorted by number of favos descending:

![image](https://user-images.githubusercontent.com/3754370/61889527-ded97000-af05-11e9-9935-e7ca37b82c9a.png)

![image](https://user-images.githubusercontent.com/3754370/61889706-337ceb00-af06-11e9-84a5-b982211a8f25.png)

**Comment by Lineflyer** [comment]
I did not yet try to reproduce, however my experience is:
- First of all you need to export the PQ in GPX1.0.1 (see settings on geocaching.com) to export the favirote points AFAIK
- Your screenshot looks like you imported the PQ without fayvx but some caches have already been known to c:geo (previously opened and still in cachecache, already stored before input) and only those have favorite points available

Offtopic:
"Der PINOMAT" is a nice and relaxed one. You might want to do this :)

**Comment by moving-bits** [comment]
@Lineflyer Thanks for your reply.

I did a cross-check on the listing which had favos - you were right, I had those saved to c:geo already.

You were right in terms of GPX format as well - mine was set to 1.0.0. Has there been a website change? IIRC I was able to set this directly in the PQ, now I have to / I can set this in my global preferences.

But anyway, I did a file compare of the two files, and GPX 1.0.1 only adds attributes, no favos. Looks like favos never have been part of the GPX export... :-(

Therefore closing this issue.

P.S.: Will have a look at "Der Pinomat" for sure (but probably not during this trip - too much way ahead of us with too much chance for time consuming traffic jams...)
